From 6ae7aed31cd7cae4d838a5d8c41df558c87414ed Mon Sep 17 00:00:00 2001
From: Joel Selvaraj <jo@jsfamily.in>
Date: Thu, 17 Jun 2021 07:43:30 +0530
Subject: [PATCH 013/211] arm64: dts: qcom: sdm845-xiaomi-beryllium: Add
 haptics support

Enable the pmi8998 SPMI haptics driver and configure the play rate for the
hardware.

[David] properties sort
Signed-off-by: Joel Selvaraj <jo@jsfamily.in>
Signed-off-by: Casey Connolly <casey@connolly.tech>
Signed-off-by: David Heidelberg <david@ixit.cz>
---
 .../arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
index 080f5294988d..28ea55177bca 100644
--- a/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
+++ b/arch/arm64/boot/dts/qcom/sdm845-xiaomi-beryllium-common.dtsi
@@ -408,6 +408,12 @@ led-1 {
 	};
 };
 
+&pmi8998_haptics {
+	qcom,wave-play-rate-us = <4878>;
+
+	status = "okay";
+};
+
 &pm8998_resin {
 	linux,code = <KEY_VOLUMEDOWN>;
 	status = "okay";
-- 
2.55.0

